buildenv

```diff
@@ -1,6 +1,9 @@
-export ZOPEN_STABLE_URL="https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.37.tar.xz"
-export ZOPEN_STABLE_DEPS="make zoslib diffutils grep sed libxml2"
+# bump: libxslt-version /LIBXSLT_VERSION="(.*)"/ https://gitlab.gnome.org/GNOME/libxslt.git|semver:*
+LIBXSLT_VERSION="1.1.37"
+
 export ZOPEN_BUILD_LINE="STABLE"
+export ZOPEN_STABLE_URL="https://download.gnome.org/sources/libxslt/${LIBXSLT_VERSION:0:3}/libxslt-${LIBXSLT_VERSION}.tar.xz"
```

Comment:
Looks like the build failed because "${LIBXSLT_VERSION:0:3}" is not compatible in posix shell

Reply:
Ah, will use the sed trick you did in another repo.